### PR TITLE
Cleaning up code snippets in readme. Removing unused param

### PR DESCRIPTION
**Pull request by gammicus**

### CODE REVIEW

File: js/simpletest.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Remember to add simple.js to your HTML scripts
-(function (root) {
+
     var simple = null;
```
Comment (nmoinvaz) [inline js/simpletest.js:3]: Remove the indent for all the code.

File: js/simpletest.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // Remember to add simple.js to your HTML scripts
-(function (root) {
+
```
Comment (nmoinvaz) [inline js/simpletest.js:2]: All the code can be unindented now.

File: README.md [hunk 3]
```diff
@@ -44,20 +44,30 @@ cmake --build .
...
+"unloadSimple": {
+    "type": "interopUnload",
+    "name": "simple",
+    "path": "{ModuleDirectory}{LibraryPrefix}simple.{LibraryExtension}"
+},
+```
+
+5. Copy *simple.js* and *simpletest.js* from the js directory to the skin directory
+6. Open *main.html* and insert the following scripts after the reference to *main.js*. Example: 
+```
```
Comment (nmoinvaz) [inline README.md]: Add the word "html" directly after ```

File: README.md [hunk 4]
```diff
@@ -44,20 +44,30 @@ cmake --build .
...
+"loadSimple": {
+    "type": "interopLoad",
+    "name": "simple",
+    "path": "{ModuleDirectory}{LibraryPrefix}simple.{LibraryExtension}"
+},
+```
+
+4. Edit *workflow.json* and add the following task to be run in the unload entry point:
+```
```
Comment (nmoinvaz) [inline README.md]: Add the word "json" directly after ```

File: README.md [hunk 5]
```diff
@@ -44,20 +44,30 @@ cmake --build .
...
-        "name": "simple",
-        "path": "{ModuleDirectory}{LibraryPrefix}simple.{LibraryExtension}"
-    },```
-5. Copy simple.js and simpletest.js from the js directory to the skin directory
-6. Open main.html and insert the following scripts after main.js: ```
-    <script src="simple.js" type="text/javascript"></script>
-    <script src="simpletest.js" type="text/javascript"></script>```
-7. Run host.exe with --disablesecurity as the first argument (during production if you sign the dll you won't need this).
+3. Edit *workflow.json* and add the following task to be run in the load entry point:
+```
```
Comment (nmoinvaz) [inline README.md]: Add the word "json" directly after ```

**Comment by gammicus** [comment]
Removed the indent in the JS file and added the code tags in readme

File: README.md [hunk 6]
```diff
@@ -44,20 +44,30 @@ cmake --build .
...
-        "name": "simple",
-        "path": "{ModuleDirectory}{LibraryPrefix}simple.{LibraryExtension}"
-    },```
-5. Copy simple.js and simpletest.js from the js directory to the skin directory
-6. Open main.html and insert the following scripts after main.js: ```
-    <script src="simple.js" type="text/javascript"></script>
-    <script src="simpletest.js" type="text/javascript"></script>```
-7. Run host.exe with --disablesecurity as the first argument (during production if you sign the dll you won't need this).
+3. Edit *workflow.json* and add the following task to be run in the load entry point:
+```JSON
```
Comment (nmoinvaz) [inline README.md]: Can you use lowercase for these? `JSON` -> `json`, `HTML` -> `html`? Is the same as how I have it in `dev` branch.

Comment (gammicus) [reply README.md]: Sure thing Mr. Monk. It's done.

**Comment by nmoinvaz** [comment]
Mr Monk approves

**Pull request merged**